identity-doctypes/src/commonMain/kotlin/com/android/identity/documenttype/knowntypes/EUPersonalID.kt

```diff
@@ -225,6 +246,7 @@ object EUPersonalID {
                 "PID holder’s gender",
                 false,
                 EUPID_NAMESPACE,
+                Icon.EMERGENCY,
```

Comment:
This mapping seems...prone to confusion. Any likelihood the icon itself will change significantly while the name of the icon does not? (meaning a new image would show up here unexpectedly)

If not, this should be fine.

Reply:
The name is defined in the Material Icon spec so guaranteed to not change. For example `emergency` is [here](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:emergency:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=emergency&icon.size=24&icon.color=%235f6368).